aerosense_tools/preprocess.py

```diff
@@ -0,0 +1,78 @@
+import numpy as np
+import pandas as pd
+from scipy.interpolate import interp1d
+from aerosense_tools.plots import plot_raw_signal
+
+
+class RawSignal:
+
+    def __init__(self, dataframe, sensor_type):
+        self.dataframe = dataframe
+        self.sensor_type = sensor_type
+
+    def plot(self, sensor_types_metadata):
```

Comment:
Please add docstrings including a description of what the method does (written in the imperative) and `:param` and `:return` fields